js/rpg_windows/Window_MenuStatus.js

```diff
@@ -43,7 +43,7 @@ Window_MenuStatus.prototype.numVisibleRows = function() {
 
 Window_MenuStatus.prototype.loadImages = function() {
     $gameParty.members().forEach(function(actor) {
-        ImageManager.loadFace(actor.faceName());
+        ImageManager.reserveFace(actor.faceName());
```

Comment:
OK, certainly we shouldn't break original behavior in this PR.
So now is the time to use the custom reservation id.
As mentioned earlier, this is caused by the special dependency relationship between `Scene_Skill, Scene_Status` and `Scene_Menu`, it's not a problem of stack.
And since this problem occurs only in two places, reservationStack is overkill.

By the way, I will send a PR to add a `reserveFace` after this is merged for the independence of the scene.